src/components/JoinUsBar/JoinUsBar.js

```diff
@@ -0,0 +1,57 @@
+import { opacify } from "polished";
```

Comment:
Ricordati di aggiungere i moduli che utilizzi nel file `packages.json` altrimenti la github action non riesce a buildare il sito

Reply:
Usa npm install per aggiungere i moduli sia al tuo ambiente locale, sia in packages.json

Reply:
Oltre a quello non penso serva questo import, lo puoi tranquillamente fare "togglando" una classe CSS fatta così
```
.opacity_half {
  opacity: 50%;
}
```